consensus/dummy/consensus.go

```diff
@@ -168,12 +168,13 @@ func (eng *DummyEngine) verifyHeaderGasFields(config *params.ChainConfig, header
 	}
 
 	// Verify BlockGasCost, ExtDataGasUsed not present before AP4
+	headerExtra := types.HeaderExtras(header)
```

Comment:
Can we rename `types.HeaderExtras` to `types.GetHeaderExtra`? I don't like Get prefixes, but I prefer it compared to having the plural of extras despite accessing a single extra (and can't name it `HeaderExtra` due to type name conflict)